components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -146,13 +183,27 @@ export class TeamDBImpl implements TeamDB {
             );
         }
 
-        const teamRepo = await this.getTeamRepo();
-        const team: Team = {
-            id: uuidv4(),
-            name,
-            creationTime: new Date().toISOString(),
-        };
-        await teamRepo.save(team);
+        let slug = slugify(name, { lower: true });
```

Comment:
I'm guessing we've ensured that the resulting slug from this function does actually conform to the regex which we're enforcing when doing the update.